libres/lib/enkf/local_ministep.cpp

```diff
@@ -156,6 +155,19 @@ void add_obsdata_node(py::handle self, LocalObsDataNode &node) {
     local_ministep_add_obsdata_node(ministep, &node);
 }
 
+LocalObsData &get_obsdata(py::handle obj) {
+    auto *ministep = reinterpret_cast<local_ministep_type *>(
+        PyLong_AsVoidPtr(obj.attr("_BaseCClass__c_pointer").ptr()));
```

Comment:
```suggestion
    auto *ministep = ert::from_cwrap<local_ministep_type>(obj);
```

Reply:
Fixed in https://github.com/equinor/ert/pull/2966.